raiden/settings.py

```diff
@@ -7,7 +7,7 @@
 RPC_CACHE_TTL = 600
 CACHE_TTL = 60
 ESTIMATED_BLOCK_TIME = 7
-GAS_LIMIT = 3141592  # Morden's gasLimit.
+GAS_LIMIT = 10 * 10**6
```

Comment:
It's all fine @palango, so I will merge but why did you increase the default gas limit? The 3.14 mill was really old a value, that's true -- ropsten now has 8 mill. But why 10? Any specific reason? I am curious.

Reply:
@palango I just saw the commit that explains that you needed bigger gas limit for the new deployment so this is why you started this. How much gas was needed exactly?

Reply:
4.8 mil last I know of - for the `TokenNetworkRegistry`.